package.json

```diff
@@ -55,7 +56,8 @@
     "checkformatting": "prettier --check '**/*.{js,scss}'",
     "format": "prettier --list-different --write '**/*.{js,scss}'",
     "test": "gulp test",
-    "version": "auto-changelog -p && git add CHANGELOG.md"
+    "version": "auto-changelog -p && git add CHANGELOG.md",
+    "push": "yarn build && yalc push"
```

Comment:
I didn't know about the `yalc push` functionality. That's useful! Docs for other developers reading this: https://github.com/wclr/yalc#pushing-updates-automatically-to-all-installations